### ISSUE TITLE
[Product] "Sold out" low contrast

### ISSUE THREAD

**Issue by svinkle** [Shopify/dawn#1246]
## Issue summary

The product template "Sold out" badge features low text contrast.

- Text: `#faf7f3`
- Background: `#f1f2ea`
- Contrast ratio: `1.05:1`

<details>
  <summary>Screenshots</summary>

  <img width="599" alt="Product template. Sold out text, barely visible, is highlighted." src="https://user-images.githubusercontent.com/1392632/150548137-a21541a7-24cc-494c-b15b-5706319efd1c.png">

</details>

#### CSS

```css
.product .price--sold-out .price__badge-sold-out {
  background: transparent;
  color: rgb(var(--color-background));
  border-color: transparent;
}
```

## Steps to reproduce

1. Check the [color contrast of the "Sold out" text](https://contrast-ratio.com/#%23faf7f3-on-%23f1f2ea).

### Expected

- Color contrast to meet AA requirements.

### Actual

- Color contrast does not meet AA requirements.

## Recommendation

Adjust the value of (or remove) the `color` property to increase the color contrast.

#### CSS

```css
.product .price--sold-out .price__badge-sold-out {
  background: transparent;
  /* color: rgb(var(--color-background)); */
  border-color: transparent;
}
```

<!-- #### Diff

```diff
- line removed
+ line added
``` -->

## Specifications

- **Dawn version:** 2.5.0
- **Browser:** Any
- **Component:** Product
- **WCAG Principal:** Perceivable
- **WCAG SC:** [1.4.3 Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)
- **Severity:** High
- **Effort:** Low
